pages/foodbar.js

```diff
@@ -0,0 +1,38 @@
+import Head from 'next/head';
+import styled from 'styled-components';
```

Comment:
styled-components is good for small changes to existing pages.  Sites that have themes, typically use a CSS system like bootstrap or tailwinds ... there are many systems.  CSS is it's own little language as you might have guessed.

Nothing wrong with what you are doing in this context, but you will want to discover a CSS system for larger projects, especially if you consider responsive design.  I use tailwinds